console/src/main/java/org/nervos/ckb/example/TransactionExample.java

```diff
@@ -0,0 +1,142 @@
+      if (temp.result == null) {
+        System.out.println(temp.error.message);
+      }
+      return temp.getTransactionHash();
+    } catch (Exception e) {
+      e.printStackTrace();
+      return "";
+    }
+  }
+
+  public Transaction generateTx(List<ReceiverBean> receiverBeanList) throws Exception {
+    long needCapacities =
+        receiverBeanList
+            .stream()
+            .mapToLong(receiverBean -> Long.valueOf(receiverBean.capacity))
+            .sum();
+    if (needCapacities < Long.valueOf(MIN_CAPACITY)) {
+      throw new Exception("Less than min capacity");
+    }
+
+    CkbSystemContract systemContract =
+        SystemContract.getSystemContract(ckbService, Network.TESTNET);
+    InputsBean inputsBean = generateInputs(inputLockScript.scriptHash(), needCapacities);
+    if (inputsBean.capacity < Long.valueOf(needCapacities)) {
+      throw new Exception("No enough Capacities");
+    }
+    List<CellOutput> cellOutputs = new ArrayList<>();
+    receiverBeanList.forEach(
+        (receiverBean -> {
+          AddressUtils addressUtils = new AddressUtils(Network.TESTNET);
+          String blake2b = addressUtils.getBlake160FromAddress(receiverBean.address);
+          cellOutputs.add(
+              new CellOutput(
+                  receiverBean.capacity,
+                  "0x",
+                  new Script(systemContract.systemScriptCellHash, Arrays.asList(blake2b))));
+        }));
+    cellOutputs.add(
+        new CellOutput(
+            String.valueOf(inputsBean.capacity - needCapacities), "0x", inputLockScript));
+    Transaction transaction =
+        new Transaction(
+            "0",
+            Arrays.asList(new OutPoint(null, systemContract.systemScriptOutPoint)),
+            inputsBean.inputs,
+            cellOutputs,
+            new ArrayList<>());
+    String txHash = ckbService.computeTransactionHash(transaction).send().getTransactionHash();
+    Witness witness = new Witness(Numeric.toBigInt(privateKeyHex), txHash);
+    inputsBean.inputs.forEach(
+        cellInput -> {
+          transaction.witnesses.add(witness);
+        });
+    return transaction;
+  }
+
+  public InputsBean generateInputs(String lockHash, long needCapacities) throws Exception {
+    List<CellInput> cellInputs = new ArrayList<>();
+    AtomicLong inputsCapacities = new AtomicLong();
+    long toBlockNumber = ckbService.getTipBlockNumber().send().getBlockNumber().longValue();
+    long fromBlockNumber = 1;
+    while (fromBlockNumber <= toBlockNumber && inputsCapacities.get() < needCapacities) {
+      long currentToBlockNumber = Math.min(fromBlockNumber + 100, toBlockNumber);
+      List<CellOutputWithOutPoint> cellOutputs =
+          ckbService
+              .getCellsByLockHash(
+                  lockHash, String.valueOf(fromBlockNumber), String.valueOf(currentToBlockNumber))
+              .send()
+              .getCells();
+      if (cellOutputs != null && cellOutputs.size() > 0) {
+
```

Comment:
1. Please do not leave blank lines after `{`.
2. Consider inserting blank line between logic trunk of code. (e.g., insert one between Ln105 and Ln106)